interface/orders/types.php

```diff
@@ -31,276 +29,377 @@
+      td1.addClass('isExpanded');
+      swapsign(td1, '+', '-');
+      $.getScript('types_ajax.php?id=' + id + '&order=<?php echo $order; ?>' + '&labid=<?php echo $labid; ?>');
+     }
+    }
 
-        // Called by the edit window to refresh a given node's children
-        function refreshFamily(id, haskids) {
-            if (id) { // id == 0 means top level
-                var td1 = $('#td' + id);
-                if (td1.hasClass('isExpanded')) {
-                    $('#con' + id).remove();
-                    td1.removeClass('isExpanded');
-                    swapsign(td1, '-', '+');
-                }
-                if (td1.hasClass('haskids') && !haskids) {
-                    td1.removeClass('haskids');
-                    // swapsign(td1, '+', '.');
-                    swapsign(td1, '+', '|');
-                    return;
-                }
-                if (!td1.hasClass('haskids') && haskids) {
-                    td1.addClass('haskids');
-                    // swapsign(td1, '.', '+');
-                    swapsign(td1, '|', '+');
-                }
-                if (haskids) {
-                    td1.parent().after('<tr class="outertr"><td colspan="5" id="con' + id + '" style="padding:0">Loading...</td></tr>');
-                    td1.addClass('isExpanded');
-                    swapsign(td1, '+', '-');
-                }
-            }
-            if (haskids)
-                $.getScript('types_ajax.php?id=' + id + '&order=<?php echo $order; ?>' + '&labid=<?php echo $labid; ?>');
-            else
-                recolor();
-        }
+    // Called by the edit window to refresh a given node's children
+    function refreshFamily(id, haskids) {
+     if (id) { // id == 0 means top level
+      var td1 = $('#td' + id);
+      if (td1.hasClass('isExpanded')) {
+       $('#con' + id).remove();
+       td1.removeClass('isExpanded');
+       swapsign(td1, '-', '+');
+      }
+      if (td1.hasClass('haskids') && !haskids) {
+       td1.removeClass('haskids');
+       // swapsign(td1, '+', '.');
+       swapsign(td1, '+', '|');
+       return;
+      }
+      if (!td1.hasClass('haskids') && haskids) {
+       td1.addClass('haskids');
+       // swapsign(td1, '.', '+');
+       swapsign(td1, '|', '+');
+      }
+      if (haskids) {
+       td1.parent().after('<tr class="outertr"><td colspan="6" id="con' + id + '" style="padding:0">Loading...</td></tr>');
+       td1.addClass('isExpanded');
+       swapsign(td1, '+', '-');
+      }
+     }
+     if (haskids)
+      $.getScript('types_ajax.php?id=' + id + '&order=<?php echo $order; ?>' + '&labid=<?php echo $labid; ?>');
+     else
+      recolor();
+    }
 
-        // edit a node
-        function enode(id) {
-            dlgopen('types_edit.php?parent=0&typeid=' + id, '_blank', 600, 500, false, 'Edit Procedure Type');
-        }
+    // edit a node
+    function enode(id) {
+     dlgopen('types_edit.php?parent=0&typeid=' + id, '_blank', 800, 750);
```

Comment:
This reverted something @sjpadgett recently did

Reply:
Changed size of popup to remove scrollbars

Reply:
See how it changes the heading on the popup with and without Jerry's changes, my code dynamically changes the heading :)